docs/xcm/building-with-xcm/xc-remote-transact.md

```diff
@@ -34,23 +33,35 @@ The sequence that we will allow will have to start like:
 This XCM sequence prefix can be followed up by arbitrary instructions, e.g. with `Transact`.
 Although we cannot guarantee this, other chains will most likely allow the same (or very similar) sequence prefix.
 
-### DescendOrigin
+#### DescendOrigin
 
 Ensures that the origin isn't a parachain but a more complex junction like `{ parachain: 2006, accountId: 0x123aff....ff }`. If this was omitted, any call would be executed as if it was sent from the *root-only* parachain's sovereign account, and we cannot allow that.
 
-### WithdrawAsset
+#### WithdrawAsset
 
 Withdraws assets on the destination chain from the derived sender account. The account must have the specified asset and the requested amount, otherwise the instruction will fail. These assets are used to pay for the XCM execution time.
 
-### BuyExecution
+#### BuyExecution
 
 Using the withdrawn assets, buys XCM executuion time.
 
-## Transact
+#### Transact
 
 Execute the specified encoded call data, without consuming weight more than specified.
 Call data can be virtually anything that is supported by the remote chain - doesn't matter what the origin chain supports.
 
+### Remote WASM Smart Contract Execution
+
+For all our runtimes supporting WASM Smart Contracts & remote transact via XCM, users can fully utilize contract uploads, instantiation and most important - calls. No special approach or setup are needed since WASM Smart Contracts are native to our chains. Sending a remote WASM SC call is same as executing any other remote transaction.
```

Comment:
```suggestion
For all our runtimes supporting WASM Smart Contracts & remote transact via XCM, users can fully utilize contract uploads, instantiation and most important - calls. No special approach or setup are needed since WASM Smart Contracts are native to our chains. Sending a remote WASM SC call is the same as executing any other remote transaction.
```